### PR TITLE
Blog Post for Open GEE 5.3.1

### PR DESCRIPTION
**Pull request by tst-nfarah**
This address #1372

Blog post:
https://tst-nfarah.github.io/earthenterprise/2019/08/05/531-release.html

### CODE REVIEW

**Comment by tst-swanson** [comment]
Only change I'd make is 
Supported Operating Systems:

- Starting **with** Open GEE 5.3.2

**Comment by tst-nfarah** [comment]
> Only change I'd make is
> Supported Operating Systems:
> 
> * Starting **with** Open GEE 5.3.2

Done

File: docs/_posts/2019-07-31-531-release.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+<br />
+
+Hi Open GEE Community!
+ 
+We are excited to announce the official release of Open GEE 5.3.1!  This release updates Open GEE with minor enhancements and bug fixes for GEE Fusion and Server.
+
+**Enhancements:**
+
+**- Uninstallation scripts are now copied to <code>/opt/google/install</code>.** This allow users to uninstall Fusion and Server without having to have a copy of the source code on hand.
```
Comment (tst-ccamp) [inline docs/_posts/2019-07-31-531-release.md]: *allows

File: docs/_posts/2019-07-31-531-release.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+**- Memory and performance improvements for large builds.** Memory utilization and performance have been improved, especially for large builds.
+
+**- Increased logging during long Fusion operations.** Fusion can now be configured to log progress while calculating overlaps between resources.
```
Comment (tst-ccamp) [inline docs/_posts/2019-07-31-531-release.md]: I think we should change "Fusion" to "System Manager"

File: docs/_posts/2019-07-31-531-release.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+**- Overlay option is now supported in terrain project command-line interfaces.** <code>genewterrainproject</code>, <code>geaddterrainproject</code> and <code>gemodifyterrainproject</code> have a new option for overlay support.
+
+**- Cutter embeds extra metadata into portable files.** The bounding box metadata can be used to improve loading performance in some applications.
+
+**Supported Operating Systems:**
+
+**- Starting with Open GEE 5.3.2 Ubuntu 14.04 will not be supported any more.** Open GEE 5.3.1 is last version supporting Ubuntu 14.04.
```
Comment (tst-ccamp) [inline docs/_posts/2019-07-31-531-release.md]: * will no longer be supported

Comment (tst-ccamp) [reply docs/_posts/2019-07-31-531-release.md]: * Open GEE 5.3.1 will be the last version to support Ubuntu 14.04 due to its recent end of life.

Comment (tst-nfarah) [reply docs/_posts/2019-07-31-531-release.md]: ok

File: docs/_posts/2019-07-31-531-release.md (hunk 2 above)
Comment (tst-nfarah) [reply docs/_posts/2019-07-31-531-release.md]: System Manager could be a too technical term. I changed it to "Fusion's system manager".

**Pull request merged**